infra/logging/src/main/java/com/simprints/infra/logging/Simber.kt

```diff
@@ -119,13 +138,51 @@ object Simber {
      * 1 kB in size."
      */
     fun tag(tag: String, isUserProperty: Boolean = false): Simber {
-        if (isUserProperty)
-            Timber.tag(USER_PROPERTY_TAG + tag)
-        else
-            Timber.tag(tag)
+        var conformingTag = tag
+
+        // Firebase Analytics requires tag to be no longer than 24 characters
+        var maxLength = 24
```

Comment:
This could be simplified a bit.

```
var conformingTag = ensureCharactersAreValid(tag)
conformingTag = if (isUserProperty) {
   ensureMaxLength(conformingTag, 24) // TODO extract const
} else {
   ensureMaxLength(USER_PROPERTY_TAG + conformingTag, 40) // TODO extract const
}
Timber.tag(conformingTag)
```